nimble/controller/src/ble_ll_rand.c

```diff
@@ -115,6 +117,7 @@ ble_ll_rand_data_get(uint8_t *buf, uint8_t len)
         if (len) {
             while ((g_ble_ll_rnum_data.rnd_size < len) &&
                    (g_ble_ll_rnum_data.rnd_size < MYNEWT_VAL(BLE_LL_RNG_BUFSIZE))) {
+            	tm_tick();
```

Comment:
this should be enabled only for BabbleSim, so perhaps there's some #define we can use to detect it?